From 64dde2f15c3b5aaee9b08e2435415b32dacd93f2 Mon Sep 17 00:00:00 2001
From: Po Liu <Po.Liu@nxp.com>
Date: Tue, 6 Mar 2018 23:29:29 +0800
Subject: [PATCH 115/706] enetc: move control BDR define to enetc.h

For sharing with TSN code.

Signed-off-by: Po Liu <Po.Liu@nxp.com>
(cherry picked from commit 2de7a464f7b5e49a2e959e5224fe845a46f89d52)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/enetc.h      | 13 +++++++++++++
 drivers/net/ethernet/freescale/enetc/enetc_cbdr.c | 13 -------------
 2 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/drivers/net/ethernet/freescale/enetc/enetc.h b/drivers/net/ethernet/freescale/enetc/enetc.h
index dea7dd4e5f97..f2a6476e7c77 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc.h
+++ b/drivers/net/ethernet/freescale/enetc/enetc.h
@@ -215,6 +215,19 @@ struct enetc_msg_cmd_set_primary_mac {
 	struct sockaddr mac;
 };
 
+#define ENETC_RING_UNUSED(R)	(((R)->next_to_clean - (R)->next_to_use - 1 \
+				 + (R)->bd_count) % (R)->bd_count)
+
+#define ENETC_CBD(R, i)	(&(((struct enetc_cbd *)((R).bd_base))[i]))
+
+#define ENETC_CBDR_TIMEOUT	1000 /* usecs */
+
+enum enetc_cbdr_stat {
+	ENETC_CMD_OK,
+	ENETC_CMD_BUSY,
+	ENETC_CMD_TIMEOUT,
+};
+
 /* SI common */
 int enetc_pci_probe(struct pci_dev *pdev, const char *name, int sizeof_priv);
 void enetc_pci_remove(struct pci_dev *pdev);
diff --git a/drivers/net/ethernet/freescale/enetc/enetc_cbdr.c b/drivers/net/ethernet/freescale/enetc/enetc_cbdr.c
index 4cd086d288b8..961211b137fe 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc_cbdr.c
+++ b/drivers/net/ethernet/freescale/enetc/enetc_cbdr.c
@@ -34,19 +34,6 @@
 
 #include "enetc.h"
 
-#define ENETC_RING_UNUSED(R)	(((R)->next_to_clean - (R)->next_to_use - 1 \
-				 + (R)->bd_count) % (R)->bd_count)
-
-#define ENETC_CBD(R, i)	(&(((struct enetc_cbd *)((R).bd_base))[i]))
-
-#define ENETC_CBDR_TIMEOUT	1000 /* usecs */
-
-enum enetc_cbdr_stat {
-	ENETC_CMD_OK,
-	ENETC_CMD_BUSY,
-	ENETC_CMD_TIMEOUT,
-};
-
 static void enetc_clean_cbdr(struct enetc_si *si)
 {
 	struct enetc_cbdr *ring = &si->cbd_ring;
-- 
2.17.1

